### DISCUSSION TITLE
Collaborateur Premium

### DISCUSSION THREAD

**Discussion by Alexandre7105** [opengisch/QField#3827] [Q&A]
Bonjour,

J'ai réalisé un QGISCLOUD grâce à QFIELDCOULD en étant le propriétaire et je souhaite inviter une collaboratrice sur ce projet hors il est marqué "Seuls les utilisateurs prémiums peuvent être ajoutés en tant que collaborateurs sur le projet privé", comment dois-je procéder pour qu'elle puisse également participer à ce projet?

Je vous remercie,

Alexandre.

**Comment by yaoyuandejuli** [comment]
Bonjour,

Il faut que tu créé une organisation puis que tu transfères ton projet dans cette organisation.
Il faut ensuite que tu ajoute les membres à ton organisation puis que tu ajoutes les membres de ton organisation au projet que tu as créé et qui se situe dans ton organisation.  

Si tu ne fais pas ça tu ne peux ajouter qu'un seul collaborateur. 

Bien cordialement,


Hugo
